guides/common/modules/proc_distributing-keys-for-remote-execution-manually.adoc

```diff
@@ -1,19 +1,19 @@
-[id="distributing-ssh-keys-for-remote-execution-manually_{context}"]
-= Distributing SSH Keys for Remote Execution Manually
+[id="distributing-script-keys-for-remote-execution-manually_{context}"]
+= Distributing Script Keys for Remote Execution Manually
 
-To distribute SSH keys manually, complete the following steps:
+To distribute Script keys manually, complete the following steps:
```

Comment:
I am unsure if this is correct. cc @adamruzicka

Reply:
It is not, there are no such things as script keys. I already pointed that out in another place and hoped it would be enough